snuba/datasets/cdc/groupassignee.py

```diff
@@ -91,4 +92,5 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
         return [
             BasicFunctionsProcessor(),
             PrewhereProcessor(),
+            SamplingRateProcessor(),
         ]
```

Comment:
It doesn't really matter, but might be good to put `SamplingRateProcessor` before `PrewhereProcessor` so that this more closely aligns with the ordering after the query type split in the future.

I wonder if that split will also make it more straightforward to have reasonable defaults and fewer ordering requirements between processors…?

Reply:
I can swap them, the result is the same, but I am not following the reasoning that `this more closely aligns with the ordering after the query type split in the future`.

Both Prewhere and SamplingRateProcessor are likely to stay on the clickhouse query side. ar at least SamplingRateProcessor msut be after we  pick the storage we run the query onto.

> I wonder if that split will also make it more straightforward to have reasonable defaults and fewer ordering requirements between processors…?

It will happen but it is not the split that makes the difference, it will be because several parameters will come from the storage that today is not accessible.

Reply:
> ar at least SamplingRateProcessor msut be after we pick the storage we run the query onto.

I didn't realize that this would be dependent on the storage. Never mind then, this belongs where it is.

> It will happen but it is not the split that makes the difference, it will be because several parameters will come from the storage that today is not accessible.

I think we are talking about different things. I meant a reasonable set of default processors, not the default parameters of those processors. But that makes sense too.